hazelcast/cluster.py

```diff
@@ -284,6 +287,17 @@ def _members_string(members):
     def _create_snapshot(version, member_infos):
         new_members = OrderedDict()
         for member_info in member_infos:
+            address_map = member_info.address_map
+            if address_map:
+                address = address_map.get(
+                    _CLIENT_ENDPOINT_QUALIFIER,
+                    address_map.get(_MEMBER_ENDPOINT_QUALIFIER, member_info.address),
+                )
+                member_info.address = address
+            else:
+                # It might be None on 4.0 servers.
+                member_info.address_map = {}
```

Comment:
In java if address map is null, memberBuilder's build() sets it to a map that includes single entry: member endpoint qualifier -> address.

Why we set it to an empty map?